From b1d7f46442e50458311573443a5c78637874f62b Mon Sep 17 00:00:00 2001
From: Anuj Mittal <anuj.mittal@intel.com>
Date: Thu, 12 Apr 2018 11:46:15 +0800
Subject: [PATCH] Makefile: check for CONFIG_IXGBEVF instead

Check for the correct config, otherwise it won't build with kbuild.

Upstream-Status: Pending

Signed-off-by: Anuj Mittal <anuj.mittal@intel.com>
---
 src/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 968ef5e..e3cb81b 100644
--- a/Makefile
+++ b/Makefile
@@ -28,7 +28,7 @@ ifneq ($(KERNELRELEASE),)
 # Makefile for the Intel(R) 10GbE PCI Express Virtual Function Driver
 #
 
-obj-$(CONFIG_IXGBE) += ixgbevf.o
+obj-$(CONFIG_IXGBEVF) += ixgbevf.o
 
 define ixgbevf-y
 	ixgbevf_main.o
